students/11th/taesukim/project/Posting/views.py

```diff
@@ -0,0 +1,78 @@
+import json
+
+from django.views import View
+from django.http  import JsonResponse
+from django.core  import serializers
+
+from User.models import User
+from .models     import Post
+from .models     import Comment
+
+class PostView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        if 'user' in data.keys() and 'img_url' in data.keys():
+            user    = data['user']
+            img_url = data['img_url']
+        else:
+            return JsonResponse({'message':'User and ImgUrl are neccesary'}, status = 400)
```

Comment:
```suggestion
            return JsonResponse({'message':'KEY_ERROR'}, status = 400)
```
꼭 있어야 하는 키가 없을 때는 키에러를 반환하시는 것이 좋습니다.